packages/eslint-plugin-svelte/src/utils/index.ts

```diff
@@ -16,6 +81,14 @@ export function createRule(ruleName: string, rule: PartialRuleModule): RuleModul
 				ruleName
 			}
 		},
-		create: rule.create as never
+		create(context: RuleContext) {
```

Comment:
Great, this is exactly what I imagined, glad to see it really was this easy (ignoring all the complexity in the called functions :D)